class AuthError(Exception):
    pass


class ParameterError(Exception):
    pass
